src/types/options.ts

```diff
@@ -98,11 +99,17 @@ interface Options {
      * Label displayed on the submit button.
      */
     submitLabel?: string;
+    titleLabel?: string;
```

Comment:
Adding comments for each option for documentation would be great! For example in this case:

```
/**
 * Label displayed on top of the cart.
 */
titleLabel?: string;
```